### DISCUSSION TITLE
Missing Traffic?

### DISCUSSION THREAD

**Discussion by ben-qnimble** [plausible/analytics#895] [Anything Else]
I'm evaluating Plausible (not self-hosted, plausible.io hosted) along with another privacy-focused analytics tracker. It seems I'm missing some traffic with Plausible. With the other tracker, I can clearly see some sessions from Safari / Mac OS that simply do not exist in Plausible. My traffic is very low, with only 0 or 1 sessions per hour, so it is easy to see what is missed with Plausible and that it is not an issue of how multiple page hits are grouped into unique visitors.

Any ideas why this would be? Does Plausible filter out bots in way that might hit legit traffic? (I'm pretty sure the missing sessions are legitimate and not crawlers.) Is there a setting to decrease the aggressiveness of this?

Thanks,

**Comment by metmarkosaric** [comment]
thanks @ben-qnimble! we do filter out bots automatically but this doesn't affect real traffic. it's really difficult to answer a question like this. so it's missing one visitor? could be someone running an adblocker for instance. if you're concerned about adblockers, you can run Plausible as a first party connection https://plausible.io/docs/custom-domain

**Reply by ben-qnimble** [reply]
An adblocker makes sense. I'll setup the custom domain and see if that makes a different. Thanks!

**Reply by metmarkosaric** [reply]
you're welcome!
